workflows/prognostic_c48_run/runtime/transformers/tendency_prescriber.py

```diff
@@ -27,55 +24,56 @@ class TendencyPrescriberConfig:
         mapper_config: configuration of mapper used to load tendency data.
         variables: mapping from state name to name of corresponding tendency in
             provided mapper. For example: {"air_temperature": "fine_res_Q1"}.
+        reference_initial_time: if time interpolating, time of first point in dataset
+        reference_frequency_seconds: time frequency of dataset
         limit_quantiles: mapping of "upper" and "lower" keys to quantile specifiers
             for limiting extremes in the Q1, Q2 dataset
     """
 
     mapper_config: loaders.MapperConfig
     variables: Mapping[str, str]
+    reference_initial_time: Optional[str] = None
+    reference_frequency_seconds: float = 900
     limit_quantiles: Optional[Mapping[str, float]] = None
 
 
 @dataclasses.dataclass
 class TendencyPrescriber:
     """Wrap a Step function and prescribe certain tendencies."""
 
-    config: TendencyPrescriberConfig
     state: DerivedFV3State
     communicator: pace.util.CubedSphereCommunicator
     timestep: float
+    variables: Mapping[str, str]
```

Comment:
Can you document this attribute with an example?

Reply:
That variable is documented with an example in the config class, seems like that's probably enough

Reply:
I added a docstring for the `TendencyPrescriber` itself, mainly so that the `time_lookup_function` has some description